src/screens/HyperSwitch/Routing/RoutingUtils.res

```diff
@@ -688,18 +688,42 @@ let validateNameAndDescription = (~dict, ~errors) => {
   })
 }
 
+let validateConditionJsonFor3ds = json => {
+  let checkValue = dict => {
```

Comment:
write this as a separate function rather than writing inside  `validateConditionJsonFor3ds`

Reply:
Separated the check value function